From 3ed2fde4946b56fcf40a130a99a6805218a9e214 Mon Sep 17 00:00:00 2001
From: Mahdi Noori <nekorawesh@gmail.com>
Date: Tue, 4 Aug 2020 15:23:56 +0300
Subject: [PATCH 6/7] tfa: eth clock from pll3q

---
 fdts/maestro-cpx.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fdts/maestro-cpx.dts b/fdts/maestro-cpx.dts
index 9a4afd756..05a90e37e 100755
--- a/fdts/maestro-cpx.dts
+++ b/fdts/maestro-cpx.dts
@@ -348,7 +348,7 @@
 	>;
 	st,pkcs = <
 		CLK_CKPER_DISABLED
-		CLK_ETH_PLL4P
+		CLK_ETH_PLL3Q
 		CLK_SDMMC12_PLL4P
 		CLK_DSI_DSIPLL
 		CLK_STGEN_HSE
-- 
2.29.2

