From c0547ce8616d8c2dca2b1907054bd448e44c363f Mon Sep 17 00:00:00 2001
From: Mahdi Noori <nekorawesh@gmail.com>
Date: Tue, 11 Aug 2020 14:30:17 +0300
Subject: [PATCH] enabling usb_ehci

---
 arch/arm/dts/maestro-cpx.dts | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm/dts/maestro-cpx.dts b/arch/arm/dts/maestro-cpx.dts
index 5b105ef7c1..43fe5b5565 100755
--- a/arch/arm/dts/maestro-cpx.dts
+++ b/arch/arm/dts/maestro-cpx.dts
@@ -1073,6 +1073,15 @@
 	/* USER CODE END usart3 */
 };
 
+&usbh_ehci{
+	status = "okay";
+
+	/* USER CODE BEGIN usbh_ehci */
+	phys = <&usbphyc_port0>;
+	phy-names = "usb";
+	/* USER CODE END usbh_ehci */
+};
+
 &usbh_ohci{
 	status = "okay";
 
-- 
2.28.0

